py/src/braintrust/merge_row_batch.py

```diff
@@ -0,0 +1,45 @@
+# Mirrors the implementation of merge_row_batch in api/chalicelib/util.py.
```

Comment:
maybe remove references to `api` etc since they're inaccessible from this codebase?